js/bundle/src/encoder.ts

```diff
@@ -23,8 +24,8 @@ export class BundleBuilder {
   private currentResponsesOffset = 0;
   private compatAdapter: CompatAdapter;
 
-  constructor(formatVersion: string = 'b2') {
-    if (formatVersion != 'b1' && formatVersion != 'b2') {
+  constructor(formatVersion: string = DEFAULT_VERSION) {
```

Comment:
WDYT about making the type of `formatVersion` in this file `'b1' | 'b2'` and moving the following code into `main`:

```ts
if (!APPROVED_VERSIONS.includes(formatVersion)) {
  throw new Error(`Invalid webbundle format version`);
}
```

Reply:
I think this is here because this constructor is used from 3 different places (outside of cli.ts), like the 2 other plugins as well. So it needs to be rechecked.

Reply:
Ah, I didn't realize that. 👍